### PR TITLE
Add support for Matomo user commands

### PR DESCRIPTION
**Pull request by kesara**
Ability to push custom commands to Matomo (#190) 
Might be a work around for #141

### CODE REVIEW

File: tests/unit/test_tag_matomo.py [hunk 1]
```diff
@@ -113,6 +113,16 @@ def test_uservars(self):
...
 
+    def test_user_commands(self):
+        context = Context({'matomo_commands': [['setIgnoreClasses', 'no-tracking'],
+                                               ['setDocumentTitle', 'foobar'],
+                                               ['tackGoal', 1, 1000.01]]})
+        r = MatomoNode().render(context)
+        for var_code in ['_paq.push(["setIgnoreClasses", "no-tracking"]);',
+                         '_paq.push(["setDocumentTitle", "foobar"]);',
+                         '_paq.push(["tackGoal", 1, 1000.01]);']:
+            assert var_code in r
```
Comment (bittner) [inline tests/unit/test_tag_matomo.py:124]: Please split this up into 3 separate assert statements. Then the actual failing test will be easier to spot. Thank you!

File: docs/services/matomo.rst [hunk 2]
```diff
@@ -109,6 +109,25 @@ in :file:`settings.py`.
...
+User commands
+-------------
+
+Matomo commands that uses ``_paq.push()`` can be passed using context variable
+``matomo_commands`` similar user variables using a list. ::
+
+    context = Context({
+        'matomo_commands': [['setIgnoreClasses', 'no-tracking'],
+                            ['setDocumentTitle', 'foobar'],
+                            ['tackGoal', 1, 1000.01]]
```
Comment (bittner) [inline docs/services/matomo.rst:121]: Maybe like this?
```suggestion
        'matomo_commands': [
            ['setIgnoreClasses', 'no-tracking'],
            ['setDocumentTitle', 'foobar'],
            ['tackGoal', 1, 1000.01],
        ]
```

**Review by bittner — CHANGES_REQUESTED** [review]
Thanks for your effort! I just had a few comments, then I think this is ready for merging.

**Comment by SilverStrings024** [comment]
I actually have this in my PR to fix #141 but for some reason git isn't letting me push.

**Comment by kesara** [comment]
@bittner Thanks for looking at my PR. Looks like #193 implements this in more generic way. So I'll close this PR.
@SilverStrings024 Can you add documentation with an example for on your PR?

**Comment by SilverStrings024** [comment]
> @bittner Thanks for looking at my PR. Looks like #193 implements this in more generic way. So I'll close this PR.
> @SilverStrings024 Can you add documentation with an example for on your PR?

Yeah, I just need to get the points Bitner hit before I can merge. If you want, you can just take the code and install the package locally then modify it to use my PR. Until I have to time to hit all Bitners points, I can't work on this.

**Comment by SilverStrings024** [comment]
> > @bittner Thanks for looking at my PR. Looks like #193 implements this in more generic way. So I'll close this PR.
> > @SilverStrings024 Can you add documentation with an example for on your PR?
> 
> Yeah, I just need to get the points Bitner hit before I can merge and I'll add in the docs before pushing next. If you want, you can just take the code and install the package locally then modify it to use my PR. Until I have to time to hit all Bitners points, I can't work on this.

**Comment by SilverStrings024** [comment]
> > > @bittner Thanks for looking at my PR. Looks like #193 implements this in more generic way. So I'll close this PR.
> > > @SilverStrings024 Can you add documentation with an example for on your PR?
> > 
> > 

Yeah, I just need to get the points Bitner hit before I can merge and I'll add in the docs before pushing next. If you want, you can just take the code and install the package locally then modify it to use my PR. Until I have to time to hit all Bitners points, I can't work on this.

**Pull request closed without merging**